example/sl-prediction/train_and_predict.py

```diff
@@ -50,9 +51,9 @@
 b_data = joblib.load(args.feature_files[1])
 
 X_A, y_A = np.array(a_data.get('X')), a_data.get('y')
-A_pairs, A_name = a_data.get('pairs'), args.names[0]
+A_pairs, A_name = np.asarray(a_data.get('pairs')), args.names[0]
 X_B, y_B = np.array(b_data.get('X')), b_data.get('y')
-B_pairs, B_name = b_data.get('pairs'), args.names[1]
+B_pairs, B_name = np.asarray(b_data.get('pairs')), args.names[1]
```

Comment:
Why do you want the pairs as an `np.array`?

Reply:
I see what you're doing later on, but my impression was that `numpy` messed up string arrays like this, so it might not play nicely later if you convert them.

Reply:
Ah, I'm not quite sure what problem this might cause but I will keep my eye out.